tests/manual-test-cases/Group5-Functional-Tests/5-22-NFS-Volume.md

```diff
@@ -0,0 +1,83 @@
+Test 5-22 - NFS Volume
+=======
+
+# Purpose:
+To verify that NFS shared volumes work with currently supported docker commands
+
+# References:
+[1 - Best practices for running VMware vSphere on NFS](http://www.vmware.com/content/dam/digitalmarketing/vmware/en/pdf/techpaper/vmware-nfs-bestpractices-white-paper-en.pdf)
+
+[2 - Docker Command Line Reference - Volume Create](https://docs.docker.com/engine/reference/commandline/volume_create/)
+
+[3 - Docker Command Line Reference - Exec](https://docs.docker.com/engine/reference/commandline/exec/)
+
+[4 - Docker Command Line Reference - Volume Inspect](https://docs.docker.com/engine/reference/commandline/volume_inspect/)
+
+[5 - Docker Command Line Reference](https://docs.docker.com/engine/reference/commandline/volume_ls/)
+
+
+# Environment:
+This test requires access to VMware Nimbus cluster for dynamic ESXi, vCenter, and NFS server creation
+
+# Test Steps:
+1. Deploy VIC appliance to vCenter with a simple cluster and one NFS server; nfs share point added here.
```

Comment:
Sure, but we should still have cases to test what happens even if our official support statement is we don't do read only and we don't do nfsv4.  We need to know that we fail correctly, because customers will def try it anyways even if we tell them no.